src/timer/timer_game_common.h

```diff
@@ -0,0 +1,205 @@
+/*
+ * This file is part of OpenTTD.
+ * OpenTTD is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 2.
+ * OpenTTD is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of the GNU General Public License along with OpenTTD. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file timer_game_common.h Definition of the common class inherited by both calendar and economy timers. */
+
+#ifndef TIMER_GAME_COMMON_H
+#define TIMER_GAME_COMMON_H
+
+#include "stdafx.h"
```

Comment:
We shouldn't include stdafx.h in header-files, I think. I think we don't do that anywhere? All cpp files already do, as first line :)